connect/core/element-template-api/src/main/java/io/miragon/miranum/connect/elementtemplate/api/ElementTemplate.java

```diff
@@ -35,5 +35,5 @@
      *
      * @return the version of the element template
      */
-    String version() default "0-1";
+    int version() default -1;
```

Comment:
kann man nicht default 'null' sagen?

Reply:
Nein leider nicht wenn ich das versuche dann kommt `Attribute value must be constant`.